CHANGES

```diff
@@ -120,6 +115,123 @@ Tools
   Verbose Compat vernacular, since these warnings can now be silenced or
   turned into errors using "-w".
 
+Bugfixes
+
```

Comment:
Good point, that could solve the problem entirely! What I searched for exactly was this:
https://coq.inria.fr/bugs/buglist.cgi?product=Coq&field0-0-0=resolution&type0-0-0=equals&value0-0-0=FIXED&field0-0-1=resolution&type0-0-1=equals&value0-0-1=INVALID&field0-0-2=resolution&type0-0-2=equals&value0-0-2=WONTFIX&field0-0-3=resolution&type0-0-3=equals&value0-0-3=DUPLICATE&field0-1-0=delta_ts&type0-1-0=lessthan&value0-1-0=2016-11-06&field0-2-0=delta_ts&type0-2-0=greaterthan&value0-2-0=2016-01-01